### PR TITLE
manifest: update for regular Zephyr upmerge 06.03.2023

### PR DESCRIPTION
**Pull request by jfischer-no**
Update the following projects:
  - zephyr (based on https://github.com/zephyrproject-rtos/zephyr/commit/6d9adf2e8af476819fce802d326a02226d98ed0c)
  - mcuboot (new-history branch)

test-sdk-nrf: testing-upmerge-2023-03-06

### CODE REVIEW

**Review by mbolivar-nordic — DISMISSED** [review]
The sdk-mcuboot changes are a history rewrite jumping straight from `v1.9.99-ncs4` to `new-history`, but they are not zero diff. This is against policy. @de-nordic did you receive an exemption from @carlescufi for this policy violation?

**Comment by carlescufi** [comment]
> The sdk-mcuboot changes are a history rewrite jumping straight from `v1.9.99-ncs4` to `new-history`, but they are not zero diff. This is against policy. @de-nordic did you receive an exemption from @carlescufi for this policy violation?

I discussed this with @de-nordic and we agreed that it is not really possible to come to a zero diff update in this case, so this is approved.

**Comment by plskeggs** [comment]
Building the `nrf_cloud_mqtt_multi_service_sample` as part of `test-fw-nrfconnect-nrf-iot_cloud` fails because `struct flash_map` no longer has an `fa_device_id` field.  The compile fails because it is logged in `subsys/net/lib/nrf_cloud/nrf_cloud_pgps.c` in the function `open_flash()`.  Printing of the `fa_device_id` field is unnecessary and can be removed.
```
	LOG_DBG("Opened flash_area: fa_id:%u, fa_device_id:%u, "
		"fa_off:%ld, fa_size:%zu, prediction_flash_area device name:%s",
		prediction_flash_area->fa_id, prediction_flash_area->fa_device_id,
		prediction_flash_area->fa_off, prediction_flash_area->fa_size, name);

**Review by adsz-nordic — APPROVED** [review]

File: samples/nrf9160/modem_shell/src/gnss/gnss.c [hunk 1]
```diff
@@ -127,7 +127,7 @@ struct event_item {
 	void    *data;
 };
 
-K_MSGQ_DEFINE(event_msgq, sizeof(struct event_item), 10, 4);
+K_MSGQ_DEFINE(gnns_event_msgq, sizeof(struct event_item), 10, 4);
```
Comment (tokangas) [inline samples/nrf9160/modem_shell/src/gnss/gnss.c:130]:
```suggestion
K_MSGQ_DEFINE(gnss_event_msgq, sizeof(struct event_item), 10, 4);
```

File: samples/nrf9160/modem_shell/src/gnss/gnss.c [hunk 2]
```diff
@@ -229,7 +229,7 @@ static void gnss_event_handler(int event_id)
 		return;
 	}
 
-	err = k_msgq_put(&event_msgq, &event, K_NO_WAIT);
+	err = k_msgq_put(&gnns_event_msgq, &event, K_NO_WAIT);
```
Comment (tokangas) [inline samples/nrf9160/modem_shell/src/gnss/gnss.c:232]:
```suggestion
	err = k_msgq_put(&gnss_event_msgq, &event, K_NO_WAIT);
```

File: samples/nrf9160/modem_shell/src/gnss/gnss.c [hunk 3]
```diff
@@ -377,7 +377,7 @@ static void data_handler_thread_fn(void)
 	struct event_item event;
 
 	while (true) {
-		k_msgq_get(&event_msgq, &event, K_FOREVER);
+		k_msgq_get(&gnns_event_msgq, &event, K_FOREVER);
```
Comment (tokangas) [inline samples/nrf9160/modem_shell/src/gnss/gnss.c:380]:
```suggestion
		k_msgq_get(&gnss_event_msgq, &event, K_FOREVER);
```

File: samples/nrf9160/modem_shell/src/cloud/cloud_lwm2m.c [hunk 4]
```diff
@@ -251,6 +251,10 @@ static void cloud_lwm2m_rd_client_event_cb(struct lwm2m_ctx *client_ctx,
 		cloud_lwm2m_rd_client_stop();
 		break;
 
+	case LWM2M_RD_CLIENT_EVENT_REG_UPDATE:
+		/* Fixme */
```
Comment (tokangas) [inline samples/nrf9160/modem_shell/src/cloud/cloud_lwm2m.c:255]:
```suggestion
		mosh_print("LwM2M: Registration update started");
```

**Review by tokangas — CHANGES_REQUESTED** [review]

**Review by koffes — DISMISSED** [review]
Please add all our changes from https://github.com/nrfconnect/sdk-nrf/pull/10559/commits

File: samples/nrf9160/modem_shell/src/gnss/gnss.c (hunk 2 above)
Comment (jfischer-no) [reply samples/nrf9160/modem_shell/src/gnss/gnss.c]: arg, forgot to fix the commit message.

**Review by tokangas — APPROVED** [review]

File: applications/serial_lte_modem/prj.conf [hunk 5]
```diff
@@ -64,6 +64,7 @@ CONFIG_IMG_MANAGER=y
 CONFIG_FLASH=y
 CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_FLASH_MAP=y
+CONFIG_STREAM_FLASH=y
 CONFIG_MPU_ALLOW_FLASH_WRITE=y
 CONFIG_IMG_ERASE_PROGRESSIVELY=y
```
Comment (junqingzou) [inline applications/serial_lte_modem/prj.conf:70]: Please add CONFIG_SLM_TFTPC=n for now
Will update SLM TFTP client with a PR later

Comment (jfischer-no) [reply applications/serial_lte_modem/prj.conf:70]: okay, disabled SLM_TFTPC.

**Review by simensrostad — APPROVED** [review]

**Review by de-nordic — COMMENTED** [review]
Tested MCUboot with MCUmgr on nrf52840dk_nrf52840. Seems to work fine.

**Review by de-nordic — APPROVED** [review]

**Comment by koffes** [comment]
Why on earth did that happen? I only re-requested a review from myself.

**Comment by joerchan** [comment]
> Why on earth did that happen? I only re-requested a review from myself.

It happens because the list is longer than the max accepted. So it works when auto-added, but when you remove, it removes the ones that exceed (from what I can tell).

**Review by hakonfam — APPROVED** [review]

**Review by SeppoTakalo — APPROVED** [review]

**Review by SeppoTakalo — APPROVED** [review]

**Review by LuDuda — APPROVED** [review]

**Comment by joerchan** [comment]
Approving TF-M and Crypto.

**Review by kszewenko — APPROVED** [review]

**Review by jorgenmk — APPROVED** [review]

**Review by emob-nordic — APPROVED** [review]

**Review by erikrobstad — APPROVED** [review]

**Review by trantanen — APPROVED** [review]

**Review by MarkusLassila — APPROVED** [review]

**Review by miha-nordic — APPROVED** [review]

**Review by sebastiandraus — APPROVED** [review]

**Review by kkasperczyk-no — APPROVED** [review]

**Review by plskeggs — APPROVED** [review]

**Pull request merged**